Testgrid: Updated docs for allow-list (#1243)
openshift/ci-tools: cmd/testgrid-config-generator/README.md

New jobs should start in `broken` until they have successive runs, then they can graduate to `informing` or `blocking`. A job does not have
to be referenced by the release controller to be informing - the release controller simply ensures it is run once per release build.

> PRs would be generated automatically for runs of the testgrid-config-generator tool which result in changes in `github.com/kubernetes/test-infra/config/testgrids/openshift`. This is done by the periodic-prow-auto-testgrid-generator job which is run once a day.

Reviewer: ```suggestion
PRs are generated automatically for runs of the testgrid-config-generator tool which result in changes in `github.com/kubernetes/test-infra/config/testgrids/openshift`. This is done by the periodic-prow-auto-testgrid-generator job which is run once a day.
```

modified:   groups.yaml
new file:   redhat-openshift-...
```

> PRs would be generated automatically for runs of the testgrid-config-generator tool which result in changes to `github.com/kubernetes/test-infra`. This is done by the periodic-prow-auto-testgrid-generator job.

Reviewer: you should put this earlier.  no one needs to build+run testgrid-config-generator at all anymore.  You can leave the instructions as they are useful if someone needs to run it manually for some reason, but make it clear they normally don't need to do any of this any more.
